scale/batch/models.py

```diff
@@ -24,6 +24,26 @@
 class BatchManager(models.Manager):
     """Provides additional methods for handling batches"""
 
+    def count_completed_job(self, batch_id):
+        """Preforms a count-plus-one on the completed job field of a Batch
+
+        :param batch_id: The unique identifier of the batch.
+        :type batch_id: int
+        """
+
+        job_count = Batch.objects.filter(id=batch_id).values('completed_job_count')
+        job_count.update(completed_job_count=job_count[0]['completed_job_count'] + 1)
```

Comment:
You need these to be a single query to prevent race conditions. Look at Django models.F(). It will allow field + 1 queries.

Also "perform" is misspelled in the docstring.